membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -13,19 +13,31 @@ import models._
 import monitoring.Metrics
 import play.api.libs.concurrent.Execution.Implicits._
 import play.api.libs.json.Json
-import play.api.mvc.{Controller, Result}
+import play.api.mvc._
 import play.filters.cors.CORSActionBuilder
 
-import scala.concurrent.Future
+import scala.concurrent.{Future, Promise}
 import scalaz.std.scalaFuture._
 import scalaz.syntax.std.option._
 import scalaz.{EitherT, \/}
+import configuration.Config.authentication
 
 
 class AttributeController extends Controller with LazyLogging {
 
+  def signed(): ActionBuilder[Request] = new ActionBuilder[Request] {
+    def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]) = {
+      if (request.headers.get("authentication").contains(authentication.key)) {
+        block(request)
+      } else {
+        Future.successful(Forbidden)
+      }
+    }
+  }
+
   lazy val corsFilter = CORSActionBuilder(Config.corsConfig)
   lazy val backendAction = NoCacheAction andThen corsFilter andThen BackendFromCookieAction
+  lazy val backendForSyncWithZuora = NoCacheAction andThen  signed andThen WithBackendFromUserIdAction
```

Comment:
as mentioned previously I see no reason to have it signed as the only malicious action would be a DOS attack which would be just as possible through any of the other endpoints that hit zuora directly, also unsigned it makes it easier for us to do an update from other tools e.g. useradmin